src/amber/cli/templates/auth.cr

```diff
@@ -18,7 +18,7 @@ module Amber::CLI
       @language = language
       @fields = fields.map { |field| Field.new(field, database: @database) }
       @fields << Field.new("email:string", hidden: false, database: @database)
-      @fields << Field.new("encrypted_password:password", hidden: false, database: @database)
+      @fields << Field.new("hashed_password:password", hidden: false, database: @database)
```

Comment:
This is more semantic, but if we want to keep it consistent with previous versions of amber, we can.

Reply:
yup, that is fine.  love it!